MediaWikiKit/MediaWikiKit/MWKDataStore.m

```diff
@@ -334,6 +334,12 @@ - (NSString*)sectionTextWithId:(NSUInteger)sectionId article:(MWKArticle*)articl
     return html;
 }
 
+- (BOOL)htmlFileExistsForSectionId:(NSUInteger)sectionId title:(MWKTitle*)title {
+    NSString* path     = [self pathForSectionId:sectionId title:title];
```

Comment:
Note: we want this to return YES even if the "Section.html" file is empty - this is because some sections *do* have zero length text strings - such as sections which have immediate sub-sections. If we didn't do this articles having such sections would *never* appear to be cached.